.travis.yml

```diff
@@ -64,6 +66,11 @@ before_install:
   # Stop build if comment contains [skip travis].
   - if $(git log -n1 --format="%B" | grep --quiet '\[skip travis\]'); then exit; fi 
   
+  - if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then brew unlink cmake; fi
+  - if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then brew install cmake; fi
+  - cmake --version
```

Comment:
I think the CMake that comes installed is 2.8.x. 
I now realized this is not what I intended. My intention was to install CMake 3.1.3 in particular. I will fix this.